scintellometry/io/arochime.py

```diff
@@ -35,9 +39,20 @@ def __init__(self, raw_files, blocksize,
         self.fedge = fedge
         self.fedge_at_top = fedge_at_top
         f = fftshift(fftfreq(nchan, (2./samplerate).to(u.s).value)) * u.Hz
-        if fedge_at_top:
+        if fedge_at_top :
             self.frequencies = fedge - (f-f[0])
-        else:
+            if self.time0.value > 1406764800  and self.time0.value < 1406937600:
+            ### reset these for just july run when we swapped a cable
+            	initfreqar = fedge - (f-f[0])
+            	newind = np.zeros(1024,dtype=int)
```

Comment:
This can be done (I think) in two lines:
```
iar = np.array([4,5,6,7,0,1,2,3])
self.frequencies = initfreqar.reshape(-1, 8)[:, iar].reshape(-1)
```